2.4-micro/2.4-micro

```diff
@@ -0,0 +1 @@
+.
```

Comment:
@phracek said in case of nginx-container that this kind of symlink breaks upstream->downstream. Not sure whether also in this case.

Reply:
What is the issue with this approach? Iirc rhcwt during syncs actually creates this symlink by default.